test/policies/dashboard_policy_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+
+class DashboardPolicyTest < ActiveSupport::TestCase
```

Comment:
Please remove unused files <3

Reply:
just for the record and future PR, all the polici_test files are empty

Reply:
same with controllers

Reply:
Let's remove them all yes :)